spec/shoryuken/middleware/server/exponential_backoff_retry_spec.rb

```diff
@@ -25,7 +25,7 @@
     it 'does not retry the job by default' do
       expect(sqs_msg).not_to receive(:change_visibility)
 
-      expect { subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise } }.to raise_error
+      expect { subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise "Error" } }.to raise_error(RuntimeError, "Error")
```

Comment:
Line is too long. [131/120]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.